client/stylesheets/_toCat.scss

```diff
@@ -5165,16 +5170,25 @@ On click the images will be displayed at normal size to complete the effect
     border-bottom: 1px solid #d3d3d7;
     padding: 30px 0;
     margin: 10px 0;
-    width: 70%;
+    width: 100%;
 }
 
+.profileSettingsLi h4 {
```

Comment:
Merge rule `.profileSettingsLi h4` with rule on line 5169<br>Selector `profileSettingsLi` should be written in lowercase with hyphens